ydb/core/protos/config.proto

```diff
@@ -1549,21 +1549,30 @@ message TTracingConfig {
         optional TSelectors Scope = 1;
         optional float Fraction = 2;
         optional uint32 Level = 3;
-        optional uint32 MaxRatePerMinute = 4;
-        optional uint32 MaxBurst = 5;
+        optional uint64 MaxRatePerMinute = 4;
+        optional uint64 MaxBurst = 5;
     }
 
     message TExternalThrottlingScope {
         optional TSelectors Scope = 1;
-        optional uint32 MaxRatePerMinute = 2;
-        optional uint32 MaxBurst = 3;
+        optional uint64 MaxRatePerMinute = 2;
+        optional uint64 MaxBurst = 3;
+    }
+
+    message TUploaderConfig {
+        optional uint64 MaxSpansPerSecond = 1;
```

Comment:
Spans per second, but rate per minute? I'd prefer common units here.

Reply:
But they limit different things, which often have different scale. Per-minute limits control traces, that often have quite some spans. But for consistency we centanly can  limit upload rate per minute as well